appender.go

```diff
@@ -13,6 +13,8 @@ import (
 	"unsafe"
 )
 
+type UUID [16]byte
```

Comment:
Thanks 👍  Would it make sense to move this to `types.go`?

Reply:
Sure. I thought since its specific to `appender` so it can be put in `appender.go` but yeah I see other such types as well.